<launch>
  <arg name="enable_option" default="-e"/>

  <node name="baxter_enable" pkg="baxter_tools" type="enable_robot.py" args="$(arg enable_option)" />

  <node name="joint_trajectory" pkg="baxter_interface" type="joint_trajectory_action_server.py"/>

  <include file="$(find baxter_moveit_config)/launch/move_group.launch" />

  <arg name="marker_size" default="4.0" />
  <arg name="max_new_marker_error" default="0.08" />
  <arg name="max_track_error" default="0.2" />
  <arg name="cam_image_topic" default="/cameras/left_hand_camera/image" />
  <arg name="cam_info_topic" default="/cameras/left_hand_camera/camera_info" />
  <arg name="output_frame" default="/left_hand_camera_axis" />

  <node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />

  <arg name="cam_image_topic_2" default="/cameras/right_hand_camera/image" />
  <arg name="cam_info_topic_2" default="/cameras/right_hand_camera/camera_info" />
  <arg name="output_frame_2" default="/right_hand_camera_axis" />

  <node name="ar_track_alvar_right" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic_2) $(arg cam_info_topic_2) $(arg output_frame_2)" />

  <!--node name="baxterball" pkg="baxterball" type="baxterball.py" respawn="true" output="screen"/ -->

</launch>

